samples/ingestion/ingestion-client/.editorconfig

```diff
@@ -0,0 +1,1362 @@
+# NOTE: Requires **VS2019 16.3** or later
+
+# Ingestion Client code rules
+# Description: Code analysis rules for Ingestion Client code
+
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# All Files
+
+[*]
+charset = utf-8-bom
```

Comment:
would you want to change that to utf-8?

Reply:
(so editorconfig and ci/check-git-head.sh do not fight each other)

Reply:
Fixed!